security-whitepaper/cloud-native-security-whitepaper.md

```diff
@@ -602,12 +604,13 @@ When considering threats specific to an organization's cloud native capabilities
 - Denial of Service (DoS) resulting from a pod that does not have resource limits applied therefore consumes the entire node level CPU and memory, worker node is then lost
 - Elevation of privilege could happen if a pod is running with unrestricted or higher privileged pod security policy or by modifying the security context of a pod or a container
 
-Threat actors to consider for cloud native security are consistent with existing threat modeling:
+Threat actors to consider for cloud native security are consistent with existing threat modeling practices:
+
+- Malicious insider - An actor with malicious intent and with authorization to perform actions within the modeled system.
+- Uninformed insider - An actor with authorization to perform actions within the modeled system (assuming anyone can be duped).
+- Malicious outsider - An actor outside of the system with malicious intent, able to launch attacks via the internet, supply chain, physical perimeter etc.
```

Comment:
Maybe worth rephrasing to clarify that the outsider does not have access to the modeled system:

`An actor outside of the system with malicious intent, able to launch attacks via the internet, supply chain, physical perimeter etc. without explicit authorization to perform actions within the modeled system`

Reply:
I'll make that change.